### PR TITLE
Change README

### PR DESCRIPTION
**Pull request by wngzhiqi**
Change ./analytics-zoo-VERSION-jar-with-dependencies.jar to $ANALYTICS_ZOO_HOME/lib/analytics-zoo-bigdl_BIGDLVERSION-spark_2.1.0-ZOOVERSION-SNAPSHOT-jar-with-dependencies.jar

### CODE REVIEW

**Comment by yangw1234** [comment]
LGTM

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ spark-submit \
 --driver-memory 200g \
 --executor-memory 200g \
 --class com.intel.analytics.zoo.examples.tfnet.Predict \
-./analytics-zoo-VERSION-jar-with-dependencies.jar --image $imagePath --model $modelPath --partition 4
+$ANALYTICS_ZOO_HOME/lib/analytics-zoo-bigdl_BIGDLVERSION-spark_2.1.0-ZOOVERSION-SNAPSHOT-jar-with-dependencies.jar --image $imagePath --model $modelPath --partition 4
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md]: remove SNAPSHOT

Comment (wngzhiqi) [reply zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md]: Now the jar name is analytics-zoo-bigdl_0.6.0-spark_2.1.0-0.2.0-SNAPSHOT-jar-with-dependencies.jar.  Need to remove SNAPSHOT?

Comment (yangw1234) [reply zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md]: SNAPSHOT is part of version?

**Comment by yangw1234** [comment]
also submit a pr to master?

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md (hunk 1 above)
Comment (hkvision) [reply zoo/src/main/scala/com/intel/analytics/zoo/examples/tfnet/README.md]: Yeah SNAPSHOT should be supposed to be within ZOOVERSION...
For a release version, there is no SNAPSHOT lol...
We need to make the README as general as possible :)

**Pull request closed without merging**